libcloud/security.py

```diff
@@ -61,6 +65,19 @@
     '/etc/ssl/certs/YaST-CA.pem',
 ]
 
+# Insert certifi CA bundle path to the front of Libcloud CA bundle search
+# path if certifi is available
+try:
+    import certifi
+except ImportError:
+    has_certifi = False
+else:
+    has_certifi = True
+
+if has_certifi and USE_CERTIFI:
+    certifi_ca_bundle_path = certifi.where()
+    CA_CERTS_PATH.insert(0, certifi_ca_bundle_path)
```

Comment:
I'd prefer having certifi as a default if it's present and not show the error message.
I'd be fine with just updating https://github.com/apache/libcloud/blob/trunk/docs/other/ssl-certificate-validation.rst if the ssl certs can be found as that is what it will point to anyways.